compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1324,20 +1339,30 @@ SmallVector<int64_t, 4> AttentionOp::getStaticLoopRanges() {
 
 SmallVector<AffineMap> AttentionOp::getIndexingMapsForOperands() {
   auto maps = getIndexingMapsArray();
-  return SmallVector<AffineMap>(maps.begin(),
-                                maps.begin() + getNumDpsInputs() - 1);
+  return SmallVector<AffineMap>(maps.begin(), maps.begin() + getNumDpsInputs());
 }
 
 SmallVector<AffineMap> AttentionOp::getIndexingMapsForResults() {
   auto maps = getIndexingMapsArray();
-  return SmallVector<AffineMap>(maps.begin() + getNumDpsInputs() - 1,
-                                maps.end());
+  return SmallVector<AffineMap>(maps.begin() + getNumDpsInputs(), maps.end());
 }
 
 //===----------------------------------------------------------------------===//
 // OnlineAttentionOp
 //===----------------------------------------------------------------------===//
 
+void OnlineAttentionOp::build(::mlir::OpBuilder &odsBuilder,
+                              ::mlir::OperationState &odsState,
+                              ::mlir::TypeRange results, ::mlir::Value query,
+                              ::mlir::Value key, ::mlir::Value value,
+                              ::mlir::Value scale, ::mlir::Value output,
+                              ::mlir::Value max, ::mlir::Value sum,
+                              ::mlir::ArrayAttr indexing_maps,
+                              std::optional<::mlir::Value> mask) {
```

Comment:
We don't need the `::mlir::` prefix here

Reply:
Done